spa/src/oauth-manager/index.tsx

```diff
@@ -0,0 +1,55 @@
+import ApiRequest from "../api";
+
+const OauthManager = () => {
+	let accessToken: string | undefined;
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	let refreshToken: string | undefined;
+	let username: string | undefined;
+	let email: string | undefined;
+
+	async function checkValidity() {
+		if (!accessToken) return;
+		const res = await ApiRequest.token.getUserDetails(accessToken);
+		username = res.data.login;
+		email = res.data.email;
+
+		return res.status === 200;
```

Comment:
If res.status !== 200, will this return `false` or will axios throw an Error?

Reply:
Added a try-catch to catch errors, in case of errors this will only return false.

Reply:
Actually where's the try/catch? I didn't see it?